### ISSUE TITLE
Custom handle error

### ISSUE THREAD

**Issue by papand13** [kamilkp/angular-sortable-view#18]
When i add a custom handle through sv-handle, i get the following error:
````
    TypeError: undefined is not a function
    at link (http://localhost:59633//js/lib/angular-sortable-view.js:491:45)
    at invokeLinkFn (http://localhost:59633//js/lib/angular.js:8111:9)
    at nodeLinkFn (http://localhost:59633//js/lib/angular.js:7623:11)
    at compositeLinkFn (http://localhost:59633//js/lib/angular.js:6991:13)
    at compositeLinkFn (http://localhost:59633//js/lib/angular.js:6994:13)
    at nodeLinkFn (http://localhost:59633//js/lib/angular.js:7618:24)
    at compositeLinkFn (http://localhost:59633//js/lib/angular.js:6991:13)
    at publicLinkFn (http://localhost:59633//js/lib/angular.js:6870:30)
    at boundTranscludeFn (http://localhost:59633//js/lib/angular.js:7009:16)
    at controllersBoundTransclude (http://localhost:59633//js/lib/angular.js:7650:18) <div class="queue-handle" sv-handle="" ng-click="toggleQueueDisplay(queue);">
````
I have the repeated list set up like this:
````
<ul id="queueList" sv-root sv-part="queues">
    <li class="queue" ng-repeat="queue in queues" sv-element>
        <div class="queue-header">
            <div class="queue-handle" sv-handle ng-click="toggleQueueDisplay(queue);">
                <div class="queue-blue-box">
                    <center><span>{{queue.queuemembers.queue_length}}</span></center>
                </div>
                <span class="queue-name">{{queue.queuemembers.name}}</span>
            </div>
            <div class="queue-toggler" ng-class="{active: queue.queuemembers.anyPhoneJoined}">
                <button ng-click="queue.queuemembers.anyPhoneJoined = !queue.queuemembers.anyPhoneJoined"><span class="queue-circle"></span></button>
            </div>
        </div>
    </li>
</ul>
````

The error occurs in this part of the plugin:
````
	module.directive('svHandle', function(){
		return {
			require: '?^svElement',
			link: function($scope, $element, $attrs, $ctrl){
				if($ctrl)
					$ctrl.handle = $element.add($ctrl.handle); // support multiple handles
			}
		};
	});
````

I could really use some help with this, as i need the handle function. Right now i cannot press any buttons on my item, because the whole thing is draggable now.

Thanks,
Jonas.

**Comment by yalamber** [comment]
got same issue. How do we add implementation of add in to jqLite?

**Comment by jdromine** [comment]
I am getting the same issue as well.  Will you fix this in the next version or should I go ahead implement my own version of add in jqLite?

**Comment by ruipenso** [comment]
+1

**Comment by kamilkp** [comment]
Fixed in v0.0.13
